build_tools/azure/render_meta.py

```diff
@@ -0,0 +1,52 @@
+import os
+import sys
+
+from jinja2 import Environment, FileSystemLoader
+
+# Constant file paths. Relative to this file, so we have to run this file from this directory
+DIST_PATH = '../../dist'
+VERSION_FILE = '../../VERSION'
```

Comment:
Shouldn't this point into the `pmdarima` dir?

Reply:
Yeah, this is another reason relative paths suuuuck. I moved this file and forgot to update that one. Will fix

Reply:
This should be fixed with 575849598b04ddd728cbb702c9e845ab00e6bad6 
 and 006a582415861cbfe9bac8104990770b316bfdce. I'll leave this unresolved so you can take a look